docker-compose.yml

```diff
@@ -62,6 +63,7 @@ services:
       - 8800:80
   postgres:
     image: "postgres:13"
+    container_name: postgres_db
```

Comment:
Is there a specific reason for this line? I think it could unnecessary collisions if you're using multiple environments (e.g. I use a dedicated one for gitifying). I believe you only reference this when configuring `pgadmin`, but it can resolve the database using `postgres` without this line.